spec/models/metadata_import_spec.rb

```diff
@@ -52,8 +52,7 @@ def initialize(id: '')
   describe '#ids' do
     it 'has ids parsed from the file' do
       expect(batchable.ids)
-        .to contain_exactly('7s75dc36z', 'wm117n96b', 'pk02c9724',
-                            'xs55mc046', 'j67313767')
+        .to contain_exactly("37720c723", "cz30ps66x", "hh63sv88v", "k0698748f", "sx61dm28w")
```

Comment:
You're using double quotes here and single everywhere else.  Also I think this is the same set of things as everywhere else, but the ordering being different makes that slightly less obvious.  I'm going to approve and merge the PR as-is, but if you have a second to change this, I'd be happy to merge that too.